ci/playbooks/collect-logs.yml

```diff
@@ -72,6 +72,7 @@
             chdir: "{{ ansible_user_dir }}/zuul-output/logs/"
             cmd: |
               cp {{ ansible_user_dir }}/*.log .;
+              cp /tmp/crio-stats.log . || true;
```

Comment:
"Add cronjob to trigger job stats" task is running in pre playbook. But this file`crio-stats.log` is not getting collected in the log directory. Below is the error from log collection (which is seen in other jobs also)
https://review.rdoproject.org/zuul/build/e4d432cbbd114c71ab9ba4a8ea2e6495/log/job-output.txt#3768

```


2023-06-27 15:59:47.841881 \| TASK [Copy logs from home directory]
--
3769 | 2023-06-27 11:59:48.267166 \| controller \| cp: target 'true;' is not a directory
3770 | 2023-06-27 15:59:48.407382 \| controller \| ERROR
3771 | 2023-06-27 15:59:48.407661 \| controller \| {
3772 | 2023-06-27 15:59:48.407743 \| controller \|   "delta": "0:00:00.007900",
3773 | 2023-06-27 15:59:48.407807 \| controller \|   "end": "2023-06-27 11:59:48.267815",
3774 | 2023-06-27 15:59:48.407863 \| controller \|   "msg": "non-zero return code",
3775 | 2023-06-27 15:59:48.407918 \| controller \|   "rc": 1,
3776 | 2023-06-27 15:59:48.407975 \| controller \|   "start": "2023-06-27 11:59:48.259915"
3777 | 2023-06-27 15:59:48.408030 \| controller \| }
3778 | 2023-06-27 15:59:48.408100 \| controller \| ERROR: Ignoring Errors


```

Reply:
aha. So it requires additional task. Thanks for sending me that